01week/rockPaperScissors.js

```diff
@@ -9,10 +9,34 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
+  hand1 = hand1.toLowerCase().trim();
+  hand2 = hand2.toLowerCase().trim();
+
```

Comment:
the code you have here doesn't run and doesn't pass the tests, please get this working and update this PR